pages/add-user-page.tsx

```diff
@@ -90,103 +85,65 @@ export const AddUserPage = (props: NavigationComponentProps) => {
       <ScrollView
         contentInsetAdjustmentBehavior="automatic"
         style={backgroundStyle}>
-        <View
-          style={{
-            backgroundColor: isDarkMode ? Colors.black : Colors.white,
-          }}>
+        <View style={{ backgroundColor: isDarkMode ? Colors.black : Colors.white }}>
+
           <Title>Cadastrar Usuário</Title>
-          <Text style={Styles.inputTitle}> Nome </Text>
-          {nameError && <Text style={Styles.errorMessage}>
-            Insira um nome válido!
-          </Text>}
-          <TextInput 
-            value={name}
-            style={Styles.userInfoContainer} 
+
+          <FormField 
+            label='Nome'
+            validationError= {nameError}
+            validationMessage='Insira um nome válido!'
             onChangeText={(n) => setName(n)}
```

Comment:
```suggestion
            onChangeText={setName}
```

Geralmente quando vc tem uma função que chama outra e passa os mesmos parâmetros (nesse caso, **o mesmo**), vc pode passar apenas a referência da função.